src/utils/actions.js

```diff
@@ -1,18 +1,25 @@
+// TODO: put in seperate file for testing. Create util fil for other functions as well?
```

Comment:
If we want to use it in different places then yes. Otherwise we could export this function for testing as well

Reply:
Can't think of other uses yet. I'll export it and test it

Reply:
We can remove the comment now I think

Reply:
yes just haven't pushed it yet